### PR TITLE
Real page

### PR DESCRIPTION
**Pull request by Hamas2000**
screen
this is a website that has information about js docmentation.

### CODE REVIEW

**Review by ahzamir — CHANGES_REQUESTED** [review]
### Hi @Hamas2000  🙋‍♀️,

> Excellent work! You have done a great job implementing the project requirements but your project still misses some of the requirements

- You have done a great job implementing all the requirements and your code passes all checks on the freecodecamp website.
- Your project has a README.md file but your file does not contain the information about your project.

# Status: Required Changes 🔁
Make changes to your code and commit and request for a re-review.

AlmostThereReachGIF

 ## Highlights 🌞
 
🎯 Great Design implemented.
image
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed and completed the Technical Documentation Page and all requirements.
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
❌Your project has a professional README.md file
✔️Your code is clean

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by Hamas2000** [comment]
@ahzamir Thank you for your review. I added a readme file.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,60 @@
...
+
+- User-friendly design suitable for all skill levels.
+- Clear and concise documentation for easy understanding.
+- Organized layout for easy navigation and quick information access.
+
+## Author 👩‍💻
+
+For inquiries or support, contact:
+- Github: [hamas2000](https://github.com/Hamas2000)
+- LinkedIn: [Hamasa Noorzai](https://www.linkedin.com/in/hamasa-noorzai)
```
Comment (ahzamir) [inline README.md]: - @Hamas2000 

- Well done Hamasa by implementing the feedback and adding the README.md file to your project:
  - This Link is broken check and solve it.

**Review by ahzamir — APPROVED** [review]
### Hi @Hamas2000 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
image

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

GreatJobThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Technical Documentation project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your project has a professional README.md file
✔️Your code is clean

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by Hamas2000** [comment]
@ahzamir  Thank you for your valuable comments. It helped my to do tasks better each time. I will merge it.

**Pull request merged**